engine/apps/api/tests/test_alert_receive_channel.py

```diff
@@ -1850,3 +1850,185 @@ def test_alert_receive_channel_webhooks_delete(
+                    "id": alert_receive_channel_to_connect_2.public_primary_key,
+                    "integration": alert_receive_channel_to_connect_2.integration,
+                    "verbal_name": alert_receive_channel_to_connect_2.verbal_name,
+                    "deleted": False,
+                },
+                "backsync": True,
+            },
+        ],
+    }
+    assert source_alert_receive_channel.connected_alert_receive_channels.count() == 2
+
+
+@pytest.mark.django_db
+def test_connected_alert_receive_channels_put(
+    make_organization_and_user_with_plugin_token,
+    make_alert_receive_channel,
+    make_alert_receive_channel_connection,
+    make_user_auth_headers,
+):
+    organization, user, token = make_organization_and_user_with_plugin_token()
+    source_alert_receive_channel = make_alert_receive_channel(organization)
+    connected_alert_receive_channel = make_alert_receive_channel(organization)
+    connection = make_alert_receive_channel_connection(source_alert_receive_channel, connected_alert_receive_channel)
+
+    # update backsync for connected integration
+    client = APIClient()
+    url = reverse(
+        "api-internal:alert_receive_channel-connected-alert-receive-channels-put",
+        kwargs={
+            "pk": source_alert_receive_channel.public_primary_key,
+            "connected_alert_receive_channel_id": connected_alert_receive_channel.public_primary_key,
+        },
+    )
+    response = client.put(url, data={"backsync": True}, format="json", **make_user_auth_headers(user, token))
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json() == {
+        "alert_receive_channel": {
+            "id": connected_alert_receive_channel.public_primary_key,
+            "integration": connected_alert_receive_channel.integration,
+            "verbal_name": connected_alert_receive_channel.verbal_name,
+            "deleted": False,
+        },
+        "backsync": True,
+    }
+
+    connection.refresh_from_db()
+    assert connection.backsync is True
+
+
+@pytest.mark.django_db
+def test_connected_alert_receive_channels_delete(
+    make_organization_and_user_with_plugin_token,
+    make_alert_receive_channel,
+    make_alert_receive_channel_connection,
+    make_user_auth_headers,
+):
+    organization, user, token = make_organization_and_user_with_plugin_token()
+
+    source_alert_receive_channel = make_alert_receive_channel(organization)
+    connected_alert_receive_channel_1 = make_alert_receive_channel(organization)
+    connected_alert_receive_channel_2 = make_alert_receive_channel(organization)
+
+    make_alert_receive_channel_connection(source_alert_receive_channel, connected_alert_receive_channel_1)
+    make_alert_receive_channel_connection(source_alert_receive_channel, connected_alert_receive_channel_2)
+
+    client = APIClient()
+    url = reverse(
+        "api-internal:alert_receive_channel-connected-alert-receive-channels-put",
```

Comment:
s/-put/-delete?

Reply:
same as https://github.com/grafana/oncall/pull/4030#discussion_r1516358997, delete is not reversible